lib/index.js

```diff
@@ -104,5 +140,4 @@ export default class Pageres {
 	}
 }
 
-objectAssign(Pageres.prototype, EventEmitter.prototype);
-objectAssign(Pageres.prototype, require('./util'));
+Object.assign(Pageres.prototype, util);
```

Comment:
This is what I found regarding Object.assign: http://also.github.io/notes/2016/03/13/flow-object-assign.html